docs/analytics.md

```diff
@@ -23,6 +23,12 @@ TAGS_PATTERN=brexit_checklist_criteria report:csv_subscriber_lists["2020-11-05"]
 LINKS_PATTERN=countries report:csv_subscriber_lists["2020-11-05"]
 ```
 
+If you want just some specific headers, you can choose which headers you want as follows:
+```bash
+HEADERS=title,individual_subscribers,daily_subscribers,weekly_subscribers,total_subscribers report:csv_subscriber_lists["2020-11-05"]
+```
+All available headers are used by default, simply pass in the name of the headers as they are in the first row of the default output
```

Comment:
Nitpick but please remove `simply`.

Reply:
Can you also add an example below on how to run the rake task equivalent please?

Reply:
👍